tensorflow/core/ops/array_ops.cc

```diff
@@ -2525,7 +2525,7 @@ REGISTER_OP("ExtractImagePatches")
     .Attr("ksizes: list(int) >= 4")
     .Attr("strides: list(int) >= 4")
     .Attr("rates: list(int) >= 4")
-    .Attr("T: realnumbertype")
+    .Attr("T: numbertype")
```

Comment:
I believe this would also include quantized types but that is not included in TF_CALL_GPU_ALL_TYPES